lib/fog/openstack/models/volume/volume.rb

```diff
@@ -42,6 +44,13 @@ def extend(size)
         def ready?
           status == 'available'
         end
+
+        def display_name
+          name
+        end
+        def display_description
+          description
+        end
```

Comment:
not sure how aliases works

you can write this shorter with:
alias_method :display_name, :name